projects/ui/src/shim.cds-core.scss

```diff
@@ -1278,23 +1278,28 @@
   --clr-signpost-content-bg-color: #{tokens.$cds-alias-object-container-background}; // var(--clr-color-neutral-0);
   --clr-signpost-content-border-color: #{tokens.$cds-alias-object-interaction-border-color}; // var(--clr-color-neutral-500);
 
-  --clr-signpost-action-icon-dimension: #{tokens.$cds-global-space-0}; // to be removed when shim is fully integrated
+  --clr-signpost-action-icon-size: #{tokens.$cds-global-space-9}; // to be removed when shim is fully integrated
   --clr-signpost-action-color: #{tokens.$cds-alias-object-interaction-color}; // var(--clr-color-neutral-600);
   --clr-signpost-action-icon-color: var(--clr-signpost-action-color);
   --clr-signpost-action-hover-color: #{tokens.$cds-alias-object-interaction-color-hover}; // var(--clr-color-action-700);
   --clr-signpost-action-active-color: #{tokens.$cds-alias-object-interaction-color-active}; // var(--clr-color-action-700);
+  --clr-signpost-padding: #{tokens.$cds-global-space-5}; // ATM - set as default
```

Comment:
This should be `-space-7` according to the spec.

Reply:
Missed that trying to make it visually better before we decided to make just the icon change. Thanks!